src/main/java/seedu/address/logic/parser/HelpCommandParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.commands.HelpConsultationCommand;
+import seedu.address.logic.commands.HelpLabCommand;
+import seedu.address.logic.commands.HelpTutorialCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new HelpCommand, HelpTutorialCommand,
+ * HelpLabCommand or HelpConsultationCommand object
+ */
+public class HelpCommandParser {
+    /**
+     * Parses the given {@code String} of arguments in the context of the HelpCommand
+     * and returns an HelpCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+
+    public HelpCommand parse(String args) {
+        if (args.equals(" tutorial")) {
```

Comment:
Here maybe we can replace the `args` by `args.strip()` to avoid additional whitespace in matching string.

Reply:
Same as above, @nerdyboy98 wrote them and I just merged them in. I think we can comment on his PR instead